packages/datadog-plugin-restify/test/index.spec.js

```diff
@@ -85,7 +85,10 @@ describe('Plugin', () => {
         it('should run handlers in the request scope', done => {
           const server = restify.createServer()
           const interval = setInterval(() => {
-            next && next() && clearInterval(interval)
```

Comment:
`clearInterval` had not been getting called, as `next()` returned undefined.

Reply:
side note, in https://github.com/rektide/dd-trace-js/commit/91e7023fb0876113f516b1f16850d9aa5e35901c i had an alternate strategy that simplified this test significantly & avoided a number of potential hazards, by switching to a `setTimeout`. however i was not sure that the intended purpose of this test ("break the async context") was satisfied to the same degree as it is with current system, where we can `setInterval` well before any handlers start to run. i think it still broke the async context & provided a clearer/more foolproof test implementation, but have gone with making more minimal changes, just in case.